src/sentry/notifications/notifications/organization_request/integration_request.py

```diff
@@ -65,6 +65,8 @@ def __init__(
     def get_context(self) -> MutableMapping[str, Any]:
         return {
             **self.get_base_context(),
+            "requester_name": self.requester.get_display_name(),
+            "organization_name": self.organization.name,
```

Comment:
How are these two values used? How does this fix the issue?

Should there be some associated test changes for this PR?

Reply:
they're here - i can try to add a test https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/emails/requests/organization-integration.html#L7-L8

Reply:
If urgent, feel free to defer it.

Cool to see how the context is used in the template! Not obvious when looking at the code.